Changelog.md

```diff
@@ -366,6 +383,10 @@ Headlining features will always be on top.
 - Fix a bunch of graphics functions to respect GR_STUB, preventing problems in standalone server
 - Clean up line endings in several files (replaced CRLF with LF basically everywhere)
 - Fixed a timestamp issue which helps lay the groundwork to add In-Game Joining to multiplayer
+- Prevent pilot and campaign files from having invalid values, by clamping them within the expected ranges when saving and loading. 
+  - Also warn player if such an invalid value was detected, telling them which setting had the wrong value and what it was reset to.
+- Try to avoid invalid types in `obj_team()`
```

Comment:
Now that I see this again, probably better to say something like, "Try to avoid assertions by preventing invalid types from being handled by `obj_team()`"